contracts/abstracts/Admin.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import { ICre8ors } from "@crea8ors/interfaces/ICre8ors.sol";
+import { IERC721Drop } from "@crea8ors/interfaces/IERC721Drop.sol";
+
+/// @title Admin
+/// @notice An abstract contract that restricts access to only the cre8orsNFT contract's admin.
+abstract contract Admin {
+    ///@notice The address of the collection contract that mints and manages the tokens.
+    address public cre8orsNFT;
```

Comment:
`optional` personally, I wouldn't store `cre8orsNFT` on the contract at all. Instead, I would follow how we're using a `target` param in `cre8ing`.

- less redundant storage.
- more reusability across the ecosystem.

<img width="810" alt="Screenshot 2023-08-02 at 1 27 45 AM" src="https://github.com/defientco/crea8ors-subscription-contracts/assets/23249402/a65c024e-646a-4213-a433-857f3447f2a9">

Reply:
you're still storing `cre8orsNFT` in contract storage right?
- expected: architecture does not require cre8orsNFT to be stored in contract storage


Am I missing something?
<img width="487" alt="Screenshot 2023-08-02 at 2 46 37 PM" src="https://github.com/defientco/crea8ors-subscription-contracts/assets/23249402/bc90afac-8e40-40eb-b57f-498dc1d92473">